Shopping.Cart.Tests/CartClearedTests.cs

```diff
@@ -6,7 +6,7 @@ namespace Shopping.Cart.Tests;
 public class CartClearedTests
 {
     [Test]
-    public async Task ClearCartTest()
+    public void ClearCartTest()
```

Comment:
I changed a few of these tests that now no longer exercise async code.  This was causing compilation warnings.